bitcoin/src/blockdata/witness.rs

```diff
@@ -38,8 +41,7 @@ pub struct Witness {
     /// like [`Witness::push`] don't have to shift the entire array.
     witness_elements: usize,
 
-    /// This is the valid index pointing to the beginning of the index area. This area is 4 *
-    /// stack_size bytes at the end of the content vector which stores the indices of each item.
+    /// This is the valid index pointing to the beginning of the index area.
```

Comment:
Perhaps add "IOW, this is the total length of all witness elements serialized (without the element count but with their sizes serialized as compact size)."